### DISCUSSION TITLE
Hide List of Field

### DISCUSSION THREAD

**Discussion by mrslnputra** [pdfme/pdfme#131] [Q&A]
Hello there!

i'm about to hide List Of Field permanently so the user just need to place the speciment which is logo in this case and cant remove that square input, any ideas? or any props that i should know?

**Comment by hand-dot** [comment]
Hi @mrslnputra 

I'll implement this issue https://github.com/pdfme/pdfme/issues/107
then you can change 'List of Field' label.
